tests/saml/test_controller.py

```diff
@@ -238,7 +238,7 @@ def test_saml_authentication_redirect(
                 })
             },
             None,
-            (create_autospec(spec=Credential), create_autospec(spec=Patron), create_patron_data_mock()),
+            (create_autospec(spec=Credential), object(), create_patron_data_mock()),
```

Comment:
Adding the `last_loan_activity_sync` dynamic property to `Patron` broke this `create_autospec` call. I don't understand the details of how `create_autospec` works but fortunately this test doesn't need a detailed mock of a Patron; it just needs something to fill the space.